packages/zosfiles/src/cli/ZosFiles.definition.ts

```diff
@@ -19,6 +19,7 @@ import { UploadDefinition } from "./upload/Upload.definition";
 import { MountDefinition } from "./mount/Mount.definition";
 import { ZosmfSession } from "../../../zosmf";
 import { UnmountDefinition } from "./unmount/Unmount.definition";
+import { HMigrateDefinition } from "./hMigrate/HMigrate.definintion";
```

Comment:
Please check the file name.

Reply:
Hey @awharn, thanks for pointing out this mistake, I will make these changes and open a new PR